tests/unit/data_strategies.py

```diff
@@ -282,14 +282,14 @@ def data_arrays_from_coords(
         x2_scaled = (x2 - min_val) / (width if width else 1.0)
     else:
         x2_scaled = 0.0
-    tmp = func(t_scaled, x1_scaled, x2_scaled)
+    tmp = func(x1_scaled, x2_scaled, t_scaled)  # t_scaled at the end
```

Comment:
Pretty sure `t_scaled` should be at the start. It is always the first one in `dims`.